server/src/inmantals/server.py

```diff
@@ -165,20 +172,31 @@ def set_env(set_var: abc.Mapping[str, Optional[str]]) -> None:
         assert isinstance(mod, module.ModuleV2), type(mod)
         logger.info(f"Module {mod.name} is v2, we will attempt to install it")
 
-        # Install all v2 modules in editable mode using the project's configured package sources
-        urls: abc.Sequence[str] = project.module_source.urls
-        if not urls:
-            raise Exception("No package repos configured for project")
-        # plain Python install so core does not apply project's sources -> we need to configure pip index ourselves
-        with env_vars(
-            {
-                "PIP_INDEX_URL": urls[0],
-                "PIP_PRE": "0" if project.install_mode == module.InstallMode.release else "1",
-                "PIP_EXTRA_INDEX_URL": " ".join(urls[1:]),
-            }
-        ):
-            logger.info("Installing modules from source: %s", mod.name)
-            project.virtualenv.install_from_source([env.LocalPackagePath(mod.path, editable=True)])
+        logger.info(f"Project {project}")
+        logger.info(f"{project.__dict__=}")
```

Comment:
What does this look like in the log?

Reply:
Currently looks like so:
```
2023-12-19 12:32:06,612 inmantals.server         INFO    Project <inmanta.module.Project object at 0x7f86dfcdd550>
2023-12-19 12:32:06,612 inmantals.server         INFO    project.__dict__={'_path': '/tmp/tmp0l15cjyv', '_metadata': ProjectMetadata(requires=[], name='Temporary project', description='Temporary project', freeze_recursive=False, freeze_operator='~=', author=None, author_email=None, license=None, copyright=None, modulepath=['/tmp/tmp0l15cjyv/libs'], repo=[], downloadpath='libs', install_mode=<InstallMode.release: 'release'>, relation_precedence_policy=[], strict_deps_check=True, agent_install_dependency_modules=False, pip=ProjectPipConfig(index_url='https://pypi.org/simple/', extra_index_url=[], pre=None, use_system_config=False)), 'name': 'Temporary project', 'project_path': '/tmp/tmp0l15cjyv', 'main_file': 'main.cf', '_ast_cache': None, 'module_source': <inmanta.module.ModuleV2Source object at 0x7f86dfd02cd0>, 'module_source_v1': <inmanta.module.ModuleV1Source object at 0x7f86e12bb110>, 'virtualenv': <inmanta.env.ActiveEnv object at 0x7f86e0083290>, 'loaded': False, 'modules': {}, 'root_ns': __root__, 'autostd': True, 'strict_deps_check': True, '_complete_ast': None}
```

I'll remove the first line (`logger.info(f"Project {project}")`)
And move the second one to debug level